sympy/physics/hydrogen.py

```diff
@@ -87,15 +92,22 @@ def Psi_nlm(n, l, m, r, phi, theta, Z=1):
     Returns the Hydrogen wave function psi_{nlm}. It's the product of
     the radial wavefunction R_{nl} and the spherical harmonic Y_{l}^{m}.
 
-    n, l, m
-        quantum numbers 'n', 'l' and 'm'
-    r
+    Parameters
+    ==========
+
+    n : integer ranging from 1 to infinity.
```

Comment:
I would suggest something like: **n : integer with possible values as 1, 2, 3, 4,..**.
The reason being, infinity is not in the domain of integers, which makes the statement "1 to infinity." misleading.

Applies for all such instances below as well.

Reply:
Corrected them.